lib/features/manage_account/presentation/profiles/identities/identities_controller.dart

```diff
@@ -163,6 +176,7 @@ class IdentitiesController extends BaseController {
     AccountId accountId, 
     CreateNewIdentityRequest identityRequest
   ) async {
+    _publicAssetsInIdentityArguments = identityRequest.publicAssetsInIdentityArguments;
```

Comment:
Why we need this assignment? Why not keep it inside states of `create` or `edit`?

Reply:
assignments removed.